hedera-node/hedera-app/src/main/java/com/hedera/node/app/HederaApp.java

```diff
@@ -85,14 +89,17 @@
             ExpiryModule.class,
             ServiceModule.class,
             QueryWorkflowModule.class,
-            HandleWorkflowModule.class
+            HandleWorkflowModule.class,
+            HederaStateModule.class
         })
 public interface HederaApp extends ServicesApp {
     /* Needed by ServicesState */
     Provider<QueryComponent.Factory> queryComponentFactory();
 
     Provider<IngestComponent.Factory> ingestComponentFactory();
 
+    Provider<WorkingStateAccessor> workingStateAccessor();
```

Comment:
This provision method can thus be,
```
WorkingStateAccessor workingStateAccessor();
```

Reply:
I see .. I need to re-look at Dagger tutorial 😐 . Thanks Michael !